workflows/funcscan.nf

```diff
@@ -10,7 +10,7 @@ def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 WorkflowFuncscan.initialise(params, log)
 
 // Check input path parameters to see if they exist
-def checkPathParamList = [ params.input, params.multiqc_config, params.annotation_bakta_db,
+def checkPathParamList = [ params.input, params.multiqc_config, params.annotation_bakta_db, params.annotation_bakta_db_light,
```

Comment:
bakta_db_light is not a path, just a boolean so doesn't need to be added here.

But note that if you change th enames, the `bakta_db` param name needs to be changed here too